docs/docs/glossary.md

```diff
@@ -342,6 +342,10 @@ A Gatsby theme is like a WordPress theme that is composable (with other themes),
 
 A [plugin](#plugin) that transforms one type of data to another. For example you might transform a spreadsheet into a [JavaScript](#javascript) array.
 
+### Transpile
+
+The process of converting code from one syntax or format to another, such as TypeScript, a superset of JavaScript which provides custom type checking during development. [Babel](#babel) is another common example of transpilation that reformats newer JavaScript code following the [ECMAScript](#ecmascript) standard to be more backwards-compatible during the site [compilation](#compilation) process.
```

Comment:
Should I go ahead and define TypeScript? 🤔 i'll leave that for another day...

Reply:
Yup, we can add that later!